docs/source/compression/pruning_transformer.rst

```diff
@@ -0,0 +1,69 @@
+Pruning Transformer with NNI
+============================
+
+Workable Pruning Process
+------------------------
+
+Here we show an effective transformer pruning process that NNI team has tried, and users can use NNI to discover better processes.
+The full code can be found `here <https://github.com/microsoft/nni/tree/v2.9/examples/model_compress/pruning/pruning_bert_glue.py>`_.
+
+In simple terms, the entire pruning process can be divided into the following steps:
+
+1. Finetune the pretrained model on the downstream task. From our experience,
```

Comment:
I think you can write them in your code. I feel a bit lost reading this.

Reply:
move to code